mne_qt_browser/_pg_figure.py

```diff
@@ -4234,6 +4232,80 @@ def __init__(self, **kwargs):
+        if not self.menuBar():
+            self.setMenuBar(QMenuBar(self))
+        menu_bar = self.menuBar()
+        mne_python = menu_bar.addMenu("MNE-Python")
+        view_menu = menu_bar.addMenu("View")
+        help_menu = menu_bar.addMenu("Help")
+        scroll_menu = view_menu.addMenu("Scroll")
+
+        def add_menu_action(menu, description, shortcut):
+            widget = QWidget()
+            layout = QHBoxLayout(widget)
+            layout.setContentsMargins(10, 2, 10, 2)
+
+            desc_label = QLabel(description)
+            shortcut_label = QLabel(shortcut)
+
+            layout.addWidget(desc_label)
+            layout.addStretch()
+            layout.addWidget(shortcut_label)
+
+            action = QWidgetAction(self)
+            action.setDefaultWidget(widget)
+            menu.addAction(action)
+
+        shortcut_data = {
+            mne_python: [
+                ("Create annotation     ", "Click + Drag"),
+                ("Mark/unmark bad channel     ", "Left-click channel"),
+                ("Mark/unmark bad segment     ", "Left-click data"),
+                ("Toggle Annotation Tool / Visibility     ", "A"),
+                ("Toggle Butterfly     ", "B"),
+                ("Toggle DC Correction     ", "D"),
+                ("Toggle Events visibility     ", "E"),
+                ("Toggle Projection Figure / all projections     ", "J"),
+            ],
+            view_menu: [
+                ("Add/remove channels     ", "Shift + Click"),
+                ("Decrease duration (¼ page)     ", "Home"),
+                ("Decrease Scale     ", "-"),
+                ("Decrease shown channels (1/10)     ", "Page Up"),
+                ("Increase duration (¼ page)     ", "End"),
+                ("Increase Scale     ", "+ / ="),
+                ("Increase shown channels (1/10)     ", "Page Down"),
+                ("Navigate time/channels     ", "Scroll"),
+                ("Open context menu     ", "Right-click plot"),
+                ("Toggle Antialiasing     ", "L"),
+                ("Toggle Overview Bar     ", "O"),
+                ("Toggle Time Format     ", "T"),
+                ("Toggle Scalebars     ", "S"),
+                ("Toggle Whitening     ", "W"),
+                ("Toggle Crosshair     ", "X"),
+                ("Toggle Zen Mode     ", "Z"),
+            ],
+            help_menu: [
+                ("Show Help     ", "?"),
+                ("Toggle Fullscreen     ", "F11"),
+                ("Close     ", "Escape"),
+            ],
+            scroll_menu: [
+                ("Scroll left (¼ page/full page)     ", "← / →"),
+                ("Scroll up (full page)     ", "↑"),
+                ("Scroll down (full page)     ", "↓"),
+            ],
+        }
+
+        for menu, items in shortcut_data.items():
+            for description, shortcut in items:
+                add_menu_action(menu, description, shortcut)
```

Comment:
You should use the same shortcuts as defined for the browser window, e.g.,

```python
    help_menu.addAction(
         'Show keyboard shortcuts',
         browser._toggle_help_fig,
         shortcut=browser.mne.keyboard_shortcuts['?']['qt_key']
     )
```

See #126